modules/ROOT/pages/retrieve-unified-schema.adoc

```diff
@@ -30,3 +30,13 @@ curl https://<GRAPH_ENDPOINT>/schema \
 -H 'client_id: <YOUR_CLIENT_ID>' \
 -H 'client_secret: <YOUR_CLIENT_SECRET>'
 ----
+
+== Download Source APIs
+
+When you download source APIs, DataGraph downloads all API sources you've added to that environment. 
+
+To download Source APIs, you need the xref:permissions.adoc[contributor or admin permission].
```

Comment:
@tomasiturralde I added these two permissions. Wasn't sure if consume permission should also be there.